### PR TITLE
inject kube token to alicloud diskplugin in the controller deployment

### PR DESCRIPTION
**Pull request by jia-jerry**
**How to categorize this PR?**

/area control-plane
/kind bug
/platform alicloud

**What this PR does / why we need it**:
AliCloud CSI need kubeconfig to create k8s go-client, though no k8s authorization are needed to be granted.
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator

```

### CODE REVIEW

File: charts/internal/shoot-system-components/charts/csi-alicloud/templates/crd-volumesnapshotclasses.yaml [hunk 1]
```diff
@@ -1,80 +0,0 @@
-{{- if semverCompare ">= 1.17" .Values.kubernetesVersion }}
```
Comment (ialidzhikov) [inline charts/internal/shoot-system-components/charts/csi-alicloud/templates/crd-volumesnapshotclasses.yaml:1]: Why you decided to remove these CRDs?

**Review by ialidzhikov — CHANGES_REQUESTED** [review]

File: charts/internal/shoot-system-components/charts/csi-alicloud/templates/crd-volumesnapshotclasses.yaml (hunk 1 above)
Comment (jia-jerry) [reply charts/internal/shoot-system-components/charts/csi-alicloud/templates/crd-volumesnapshotclasses.yaml:1]: I supposed they were not used. See [here](https://github.com/gardener/gardener-extension-provider-alicloud/blob/master/pkg/controller/controlplane/valuesprovider.go#L217). Did I make some mis-understanding?

File: charts/internal/shoot-system-components/charts/csi-alicloud/templates/auth/csi-attacher-rbac.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ metadata:
 rules:
 - apiGroups: [""]
   resources: ["persistentvolumes"]
-  verbs: ["get", "list", "watch", "update"]
+  verbs: ["get", "list", "watch", "update", "patch"]
```
Comment (jia-jerry) [inline charts/internal/shoot-system-components/charts/csi-alicloud/templates/auth/csi-attacher-rbac.yaml:19]: It seems wired those `patch` verbs are needed when projected token feature is enabled. 
The verbs should be required according to [csi-attacher](https://github.com/kubernetes-csi/external-attacher/blob/v3.3.0/pkg/controller/csi_handler.go#L786). But it worked before without this verb.
The same behavior occured to volumeattachments. It is interesting. @rfranzke  @ialidzhikov

File: charts/internal/shoot-system-components/charts/csi-alicloud/templates/crd-volumesnapshotclasses.yaml (hunk 1 above)
Comment (jia-jerry) [reply charts/internal/shoot-system-components/charts/csi-alicloud/templates/crd-volumesnapshotclasses.yaml:1]: OK. BTW, how long do you plan to keep the legacy one?

**Comment by jia-jerry** [comment]
Till now, this PR fixed all the CSI issue when projected-token-mount is used. Would you please review it?

**Review by rfranzke — APPROVED** [review]
Well done, thanks!

/lgtm

**Pull request merged**